### ISSUE TITLE
add-on directory

### ISSUE THREAD

**Issue by polyester** [plone/ploneorg.core#23]
This should be a link to the paragon.plone.org project (when finished) and clever links to pypi.

**Comment by 3dogMcNeill** [comment]
Should this be required for site launch (having paragon.plone.org ready)? Because it does also need to be included on plone.com.

**Comment by polyester** [comment]
Yep, we do need this before launch. It's not hard, just needs doing, and it's a good way of chasing the jury so it finally gets done.
Note: paragon.plone.org will only have the top 50 (for a random value close to 50...)  add-ons, the rest is just some long query-URL's to pypi. Those are quite easy, actually.

**Comment by tkimnguyen** [comment]
It's good to have a hard target for paragon.plone.org :)

**Comment by 3dogMcNeill** [comment]
Great!

**Comment by tkimnguyen** [comment]
@vangheem is also interested in creating a view of Plone-tagged pypi packages

**Comment by allcaps** [comment]
This is the Django packages overview: http://www.djangopackages.com 
The site is open source: https://github.com/pydanny/djangopackages

**Comment by tkimnguyen** [comment]
(also posted at https://community.plone.org/t/paragon-vs-ploneawesome/680/15?u=tkimnguyen) 

For the time being, the https://plone.org/download/add-ons/popular-add-ons page, which you get to from https://plone.org/download , is a curated list that came out of Paragon v1.

You can recommend additions to the list by filing an issue at the new repo https://github.com/plone/paragon.addons/issues

Let's try this and see what happens.
